frontend-project/scripts/generate_interface.js

```diff
@@ -21,6 +21,9 @@ const main = async () => {
         type = 'number';
       } else if (pvalue.type == 'boolean') {
         type = 'boolean';
+      } else if (typeof pvalue.type === 'undefined' && pvalue['$ref']) {
+        const name = pvalue['$ref'].replace('#/definitions/', '');
+        type = `${name}`;
```

Comment:
Kierunek dobry, albo ja czegoś nie widzę, albo w praktyce tego nie wykorzystujesz i wygenerowane interfejs nie uwzględnia tego.

Reply:
Chodzi o sytuacje gdy otrzymujemy obiekt w odpowiedzi a nie tablicę. Np. jak były warunkowe serializery to wystąpiła sytacja gdzie pracowałem nad przesyłaniem np. części Case (id i nazwa) i było to pod ref. CaseAutocomplete. I o ile było to tablicy to bez problemu, natomiast była refka do elementu to niepoprawnie budowało definicje.